### PR TITLE
CIV-14664 CCD PR to update Judgments Tab in ExUI

### PR DESCRIPTION
**Pull request by sjamihmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-14664

### Change description ###
CCD PR to update Judgments Tab in ExUI

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: ccd-definition/CaseTypeTab/DefaultJudgmentDetailsDJSpec.json [hunk 1]
```diff
@@ -1,24 +1,23 @@
 [
   {
     "CaseTypeID": "CIVIL",
```
Comment (Azam-Hmcts) [inline ccd-definition/CaseTypeTab/DefaultJudgmentDetailsDJSpec.json:3]: You cant do changes directly on production file. There should be prod versin for it. Could we keep same TabId?

Comment (sjamihmcts) [reply ccd-definition/CaseTypeTab/DefaultJudgmentDetailsDJSpec.json]: Done

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldDJspec-nonprod.json
Comment (hmcts-version1-ignacio) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldDJspec-nonprod.json:1]: Shouldn't these files also be under "JO"?

Comment (sjamihmcts) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldDJspec-nonprod.json:1]: Actually we need to check for this field: fully PaymentDate as part of DJ flow i.e 
Default Judgment -> Judgment Paid in Full

File: ccd-definition/CaseTypeTab/DefaultJudgmentDetailsDJSpec-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,102 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
```
Comment (Azam-Hmcts) [inline ccd-definition/CaseTypeTab/DefaultJudgmentDetailsDJSpec-nonprod.json:3]: Do we need this nonprod.json file?

**Review by hmcts-version1-ignacio — APPROVED** [review]

File: ccd-definition/CaseTypeTab/DefaultJudgmentDetailsDJSpec-nonprod.json (hunk 2 above)
Comment (sjamihmcts) [reply ccd-definition/CaseTypeTab/DefaultJudgmentDetailsDJSpec-nonprod.json]: Removed this file as its not needed anymore

**Review by hmcts-version1-ignacio — DISMISSED** [review]

**Review by Omaira-Melo-Hmcts — APPROVED** [review]

**Review by kalachandrasekar1 — DISMISSED** [review]

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField-JO-nonprod.json [hunk 3]
```diff
@@ -1,8 +1,129 @@
 [
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "joDefendantName1",
+    "UserRoles": [
+      "caseworker-civil-solicitor",
+      "caseworker-civil-admin",
+      "caseworker-civil-staff"
+    ],
+    "CRUD": "CRU"
```
Comment (sherlynkhaw) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseField-JO-nonprod.json:10]: if these are going to be fields that are shown only on the judgement tab, can they be R only?

Comment (sjamihmcts) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseField-JO-nonprod.json:10]: yes, think so. Will update them to R.

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by Omaira-Melo-Hmcts — APPROVED** [review]

**Review by Azam-Hmcts — APPROVED** [review]

**Pull request merged**